src/index.ts

```diff
@@ -1,10 +1,10 @@
-function parseMatter(s: string): any {
-  if (s.trim().startsWith("{")) {
+function parseMatter(s: string) {
+  if (s.trim().startsWith('{')) {
     // JSON
     return JSON.parse(s);
   }
-  // TODO support YAML
-  return {};
+  const yaml = require('js-yaml');
```

Comment:
a really huge dep watch out

Reply:
I'll check for a light one

Reply:
there was none at this time that why I didn’t want to use it

Reply:
Lightest I found:
![2021-08-04_17-49](https://user-images.githubusercontent.com/1789748/128212687-579ea1da-0e6b-4e16-bf8f-4b0d4bf65ab4.png)

Reply:
maybe but looked like unstable to me

Reply:
1M weekly why not